content/techniques/events.md

```diff
@@ -79,26 +101,16 @@ handleOrderEvents(payload: OrderCreatedEvent | OrderRemovedEvent | OrderUpdatedE
 }
 ```
 
-#### Dispatching events
-
-To dispatch an event, first inject `EventEmitter2` using standard constructor injection:
-
-```typescript
-constructor(private eventEmitter: EventEmitter2) {}
-```
-
-> info **Hint** Import the `EventEmitter2` from the `@nestjs/event-emitter` package.
+Note that such a wildcard only applies to one block. The argument `order.*` will match, for example, the events `order.created` and `order.shipped` but not `order.delayed.out_of_stock`. In order to listen to such events,
+use the `multilevel wildcard` pattern (i.e, `**`), described in the `EventEmitter2` package.
```

Comment:
Any chance you can link to `EventEmitter2` documentation (GitHub, docs site, whatever they have)?